deepchem/models/torch_models/ferminet.py

```diff
@@ -205,10 +226,12 @@ def __init__(self,
         Torch tensor containing electrons for each atom in the nucleus
     molecule: ElectronSampler
         ElectronSampler object which performs MCMC and samples electrons
-    loss_value: Optional[torch.Tensor] (default None)
+    loss_value: torch.Tensor (default torch.tensor(0))
         torch tensor storing the loss value from the last iteration
-    pretraining_loss_list: List (default [])
-        list with losses for every epoch
+    running_diff: torch.Tensor
```

Comment:
running diff keeps track of the running average of losses for every sampling move in an epoch. resets to 0 after every epoch